scripts/component_readme/generate_readme.py

```diff
@@ -24,6 +24,8 @@ def generate_readme(component_spec: ComponentSpec, *, component_dir: Path) -> st
         description=component_spec.description,
         consumes=component_spec.consumes,
         produces=component_spec.produces,
+        is_consumes_generic=component_spec.is_generic("consumes"),
```

Comment:
What do you think about renaming `is_generic` to `accept_additional_fields`. We are taking steps away of the proper generic components? In my opinion we should try to avoid the usage of naming `generic`. Then the documentation would be align with the internal naming too.